sdk/containerregistry/Azure.Containers.ContainerRegistry/src/autorest.md

```diff
@@ -8,3 +8,11 @@ require:
     -  https://github.com/Azure/azure-rest-api-specs/tree/255757f41275e8ec474361690ea8886cae8a503b/specification/containerregistry/data-plane/readme.md
 model-namespace: false
 ```
+
```

Comment:
We have work in progress to update to a swagger to generate much of the Track 2 library: https://github.com/azure/azure-sdk/issues/2478